systemtest/src/main/java/io/strimzi/systemtest/resources/operator/HelmInstallation.java

```diff
@@ -24,29 +24,27 @@
 /**
  * Class for handling the Helm installation via the Helm client.
  */
-public class HelmInstallation implements InstallationMethod {
+public record HelmInstallation(ClusterOperatorConfiguration clusterOperatorConfiguration) implements InstallationMethod {
```

Comment:
Why is `HelmInstallation` only class from the tree classes that is changed to `record`? That breaks the consistency which I wanted to have between classes.

Reply:
Ok, I undid that change.